Package.swift

```diff
@@ -27,12 +27,13 @@ let package = Package(
         )
     ],
     dependencies: [
-      .package(url: "https://github.com/IBM-Swift/CircuitBreaker.git", .upToNextMajor(from: "3.0.0"))
+        .package(url: "https://github.com/IBM-Swift/CircuitBreaker.git", .upToNextMajor(from: "3.0.0")),
```

Comment:
@ShihabMehboob We just released a newer version of CircuitBreaker (5.0.0). Can you update Package.swift accordingly so we can leverage that version in this PR?